apps/web/public/static/locales/en/common.json

```
@@ -2233,6 +2233,8 @@
   "add_a_redirect": "Add a redirect",
   "create_entry": "Create entry",
   "time_range": "Time range",
+  "automatically_add_to_all_team_members": "Automatically add to all team members",
```

```suggestion
  "automatically_add_to_all_team_members": "Automatically add all team members",
```
I would prefer that, but I don't know if we have any official figma designs for that?